src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -611,4 +670,50 @@ public String toString() {
       return name().toLowerCase(Locale.ROOT);
     }
   }
+
+  /* https://www.confluent.io/blog/kafka-connect-deep-dive-error-handling-dead-letter-queues/ */
+  public enum ErrorTolerance {
+
+    /** Tolerate no errors. */
+    NONE,
+
+    /** Tolerate all errors. */
+    ALL;
+
+    public String value() {
+      return name().toLowerCase(Locale.ROOT);
+    }
+
+    /* Validator to validate behavior.on.null.values which says whether kafka should keep null value records or ignore them while ingesting into snowflake table. */
+    public static final ConfigDef.Validator VALIDATOR =
+        new ConfigDef.Validator() {
+          private final ConfigDef.ValidString validator = ConfigDef.ValidString.in(names());
+
+          @Override
+          public void ensureValid(String name, Object value) {
+            if (value instanceof String) {
```

Comment:
What happens if it's not a string?

Reply:
This validator usually runs when we pass in a config value which is already in form of json and each values are strings. There is technically no need to check this but I followed how other `ensureValids` were written.